packages/block-library/src/buttons/edit.native.js

```diff
@@ -0,0 +1,163 @@
+/**
+ * External dependencies
+ */
+import { View } from 'react-native';
+/**
+ * WordPress dependencies
+ */
+import {
+	InnerBlocks,
+	BlockControls,
+	AlignmentToolbar,
+} from '@wordpress/block-editor';
+import { withSelect, withDispatch } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+import { createBlock } from '@wordpress/blocks';
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { name as buttonBlockName } from '../button/';
+import styles from './editor.scss';
+
+const ALLOWED_BLOCKS = [ buttonBlockName ];
+const BUTTONS_TEMPLATE = [ [ 'core/button' ] ];
+const ALIGNMENT_MAP = {
+	left: 'flex-start',
+	center: 'center',
+	right: 'flex-end',
+};
```

Comment:
How about extract this to stylesheet as below:

```
.is-aligned-left {
	justify-content: flex-start;
}

.is-aligned-center {
	justify-content: center;
}

.is-aligned-right {
	justify-content: flex-end;
}

```

and then pull it like this:

```
styles[ `is-aligned-${ align }` ]
```